EnglishFilter/sections/general_extensions.txt

```diff
@@ -232,6 +232,8 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/75613
+mp3offline.org#%#//scriptlet("prevent-window-open")
```

Comment:
It seems that it "breaks" share buttons, so I will add a different rule.

Reply:
Ah yes. Thanks.